src/qiskit_sphinx_theme/assets/styles/_custom-directives.scss

```diff
@@ -73,13 +74,23 @@
 
   opacity: 0.7;
 
+  @media (max-width: 67em) {
+    width: 15rem;
+  }
+
+  @media (max-width: 40em) {
+    width: 10rem;
+  }
+
   // Float the image to the right of the container.
+  // Center it vertically if the image isn't tall enough.
   display: flex;
+  align-self: center;
   align-items: center;
   justify-content: flex-end;
 
   img {
-    max-height: 100%;
+    object-fit: contain;
     max-width: 100%;
   }
```

Comment:
Another suggestion is keeping the image full height and cropping a bit (I think `object-fit: cover;` would do it but haven't tried).

Reply:
Thanks for the suggestion!

I personally think cropping would be a worse user experience. For example, this is a qiskit-card in Terra. It would be a brand violation to crop the Qiskit logo:

![Screenshot 2023-07-03 at 11 12 55 AM](https://github.com/Qiskit/qiskit_sphinx_theme/assets/14852634/daf3c10d-8987-4662-a6b2-45925615986d)

Reply:
Ok fair enough! Let's leave it as it is